aeolus/aeolus.h

```diff
@@ -1,21 +1,21 @@
-//=============================================================================
-//  MusE Score
-//  Linux Music Score Editor
+// =============================================================================
+// MusE Score
+// Linux Music Score Editor
 //
-//  Copyright (C) 2002-2010 Werner Schweer and others
+// Copyright (C) 2002-2010 Werner Schweer and others
 //
-//  This program is free software; you can redistribute it and/or modify
-//  it under the terms of the GNU General Public License version 2.
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 2.
 //
-//  This program is distributed in the hope that it will be useful,
-//  but WITHOUT ANY WARRANTY; without even the implied warranty of
-//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-//  GNU General Public License for more details.
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
 //
-//  You should have received a copy of the GNU General Public License
-//  along with this program; if not, write to the Free Software
-//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
-//=============================================================================
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+// =============================================================================
```

Comment:
I wonder whether it makes sense to change spaces for all these comments though? Or is it not configurable with `uncrustify`?

Reply:
I did not find settings to disable this

Reply:
Looks like it may be related to [this parameter](https://github.com/uncrustify/uncrustify/blob/c7e1f366bab5496e79ef1939b295dc8071de1d9c/documentation/htdocs/default.cfg#L861)

Reply:
Thanks, turn this parameter to ignore